internal/pkg/cli/app_init.go

```diff
@@ -192,9 +191,9 @@ Deployed resources (such as your service, logs) will contain this app's name and
 // If the user chooses to enter a custom path, then we prompt them for the path.
 func (opts *InitAppOpts) askDockerfile() error {
 	// TODO https://github.com/aws/amazon-ecs-cli-v2/issues/206
-	dockerfiles, err := opts.listDockerfileSelections()
+	dockerfiles, err := validateDockerfiles(opts.fs)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to find Dockerfile: %w", err)
```

Comment:
It looks like all error scenarios in the `validateDockerfiles` function are already wrapping the error, so I don't think we need to wrap again here.

Reply:
I wrap up the error message here is to be consistent with the error massage below (failed to select Dockerfile). Also, I think adding another wrapped-up error message layer here can effective help user to identify the location where they fail ("read directory fail:" might not be that intuitive, but "failed to find Dockerfile: read directory fail: some error" lets them know that they fail because Dockerfile cannot be found).